ote/src/cljc/ote/time.cljc

```diff
@@ -24,6 +25,24 @@
          cljs-time/to-default-time-zone
          (format/unparse (format/formatter "dd.MM.yyyy HH:mm"))))))
 
+#?(:cljs
+   (defn js-time-to-string [time]
+     (if  (nil? time)
+       " " ;: if nil - print empty string
```

Comment:
not an empty string, contains a single space.

Reply:
Nice catch. Thanks.